### DISCUSSION TITLE
Invalid object name Subscription use .createSubscriptionType() instead

### DISCUSSION THREAD

**Discussion by PatricioNG** [hayes/pothos#1210] [Q&A] [answered]
Is there a way to get around the error of names conflicting for a `PrismaObject`? I have a model defined in the database as `Subscription` so I'd like to preserve the name, ie:

```
const Subscription = Builder.prismaObject('Subscription', {
	name: 'Subscription',
	fields: (t) => ({
        //...Rest of the object
```

But this throws an error:

```
PothosSchemaError [GraphQLError]: Invalid object name Subscription use .createSubscriptionType() instead
```

@pothos/core": "^3.41.0" if needed

**Comment by hayes** [comment, marked as answer]
There isn't anything built into Pothos to work around this.  Pothos doesn't currently support re-naming to root types (Query, Mutation, Subscription), which would be a prerequisite to removing that check.

I would recommend avoiding using Subscription as the name in your graphql schema for something other than graphql subscriptions.  

The workaround would be to change the name on the built schema, but that's a pretty hacky solution

**Reply by PatricioNG** [reply]
Gotcha, thanks @hayes! Appreciate the speedy reply. 👏🏻
